packages/flutter_image/lib/network.dart

```diff
@@ -28,12 +28,13 @@ import 'package:flutter/widgets.dart';
 class NetworkImageWithRetry extends ImageProvider<NetworkImageWithRetry> {
   /// Creates an object that fetches the image at the given [url].
   ///
-  /// The arguments must not be null.
-  const NetworkImageWithRetry(
-    this.url, {
-    this.scale = 1.0,
-    this.fetchStrategy = defaultFetchStrategy,
-  });
+  /// The arguments must not be null, except for [requestHeaders] which can
+  /// be null if no customer request headers are needed.
```

Comment:
Please instead document what the new parameters mean.

Reply:
Updated documentation.